optimizely/marketing-website
# Remove all typography fonts from Marketing Website

1) Remove typography style tag from wrapper.hbs
2) Remove all references to typography variables in stylesheets
3) Remove all typography variables
4) Remove the fonts folder
5) All proxima all the time

To test:
Click around on a bunch of pages and check that no fonts are broken. Check the console to verify typography is not loading. I'm not really sure what removing the `fonts` directory will do but I don't see why we need it.
